server/controllers/routes/post/put.js

```diff
@@ -0,0 +1,65 @@
+const { join } = require('path');
+
+const { eventSchema, publicServiceSchema } = require('../../utils/postSchema');
+const {
+  updateEventQuery,
+  updateTopicQuery,
+  updatePublicServiceQuery,
+  updateSecondaryTagQuery,
+} = require('../../../database/queries/updatePost');
+
+const updateEvent = (req, res, next) => {
```

Comment:
you should ensure if the post belong to login-ed user 
i mean in your case any user can edit any post and that impossible